pkg/common/constant.go

```diff
@@ -0,0 +1,10 @@
+package common
+
+const (
+	ErrorDetailsInfoLogLevel = iota
+	DefaultInfoLogLevel
+	StatesInfoLogLevel
+	StatesDetailsInfoLogLevel
+	DebugInfoLogLevel
+	TraceInfoLogLevel
+)
```

Comment:
link to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md so we know what the reference is supposed to be.